index.d.ts

```diff
@@ -121,8 +121,50 @@ declare namespace boxen {
 		Align the text in the box based on the widest line.
 
 		@default 'left'
+		@deprecated Use `textAlignement` instead.
 		*/
 		readonly align?: 'left' | 'right' | 'center';
+
+		/**
+		Align the text in the box based on the widest line.
+
+		@default 'left'
+		*/
+		readonly textAlignement?: 'left' | 'right' | 'center';
+
+		/**
+		Display a title on the top of the box.
```

Comment:
```suggestion
		Display a title at the top of the box.
```

Reply:
My bad on that one, please let me know if I committed other english mistakes.